docs/almaSublibraryCode2Isil.sh

```diff
@@ -18,13 +18,13 @@ cd ../data/almaSublibraryCode2Isil
 rm $GENERATED_FILE.tmp
 
 for fn in $(ls); do
-	echo $fn
+	echo "$fn"
 	IFS=$'\n' # read whole lines
-	for rows in $(cat $fn); do
-			echo "zeile: $rows"
-			libraryCode=$(echo $rows | cut -f1)
-			isil=$(echo $rows | cut -f2)
-			echo "${MAIN_LIBRARY_CODE}$(echo $fn|cut -d '.' -f1)+${libraryCode}	${isil}" >> $GENERATED_FILE.tmp
+	for rows in $(cat "$fn"); do
+			echo zeile: "$rows"
+			libraryCode=$(echo "$rows" | cut -f1)
+			isil=$(echo "$rows" | cut -f2)
+			echo ${MAIN_LIBRARY_CODE}$(echo "$fn"|cut -d '.' -f1)+""${libraryCode}"	"${isil}"" >> $GENERATED_FILE.tmp
```

Comment:
The previous version was better. The new double quotation marks are wrong. Just add them around `$fn`:

```suggestion
			echo "${MAIN_LIBRARY_CODE}$(echo "$fn" | cut -d '.' -f1)+${libraryCode}	${isil}" >> $GENERATED_FILE.tmp
```

Reply:
Thx! Did so in https://github.com/hbz/lookup-tables/pull/57/commits/2f740527e1d0d8dab6105b3034f3a58a832a54fc.